client/my-sites/earn/components/add-edit-plan-modal/index.tsx

```diff
@@ -106,8 +91,10 @@ const RecurringPaymentsPlanAddEditModal = ( {
 		product?.buyer_can_change_amount ?? false
 	);
 
+	const currencyList = Object.keys( connectedAccountMinimumCurrency );
+
 	const defaultCurrency = useMemo( () => {
-		const flatCurrencyList = currencyList.map( ( e ) => e.code );
+		const flatCurrencyList = currencyList.map( ( e ) => e.toUpperCase() );
```

Comment:
Same here...  [They seem to be](https://github.com/Automattic/jetpack/blob/55861aaa63d4cde01974afc51b03d0a6ed03075b/projects/plugins/jetpack/modules/memberships/class-jetpack-memberships.php#L107) in uppercase already... And `currencyList` is already a flat list...

Reply:
Fixed.